<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QZPlotwidget</class>
 <widget class="QWidget" name="QZPlotwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>941</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="plotTitle">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>10</y>
     <width>941</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Graphs</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="xMinIF">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>740</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xMaxIF">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>740</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yMinIF">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>710</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yMaxIF">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="toolsGroupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>131</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string>Tools</string>
   </property>
   <widget class="QCheckBox" name="logScaleCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log. Scale</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="allowDragCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow Drag.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recenterPB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Recenter</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dragXCB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X Axis</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dragYCB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string>Y Axis</string>
    </property>
   </widget>
   <zorder>logScaleCB</zorder>
   <zorder>recenterPB</zorder>
   <zorder>dragXCB</zorder>
   <zorder>dragYCB</zorder>
   <zorder>allowDragCB</zorder>
  </widget>
  <widget class="QWidget" name="yAxisHitbox" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>51</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="xAxisHitbox" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>690</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
  </widget>
  <zorder>plotTitle</zorder>
  <zorder>xMinIF</zorder>
  <zorder>xMaxIF</zorder>
  <zorder>yMinIF</zorder>
  <zorder>yMaxIF</zorder>
  <zorder>toolsGroupBox</zorder>
  <zorder>customPlot</zorder>
  <zorder>xAxisHitbox</zorder>
  <zorder>yAxisHitbox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
